### PR TITLE
TorXakis interface

### PR DESCRIPTION
**Pull request by keremispirli**
Wrapping TorXakis functionality with a library to provide programming interface.
Build a web server that uses this library.

This PR is for review and QA support during development; do not merge yet.

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -58,12 +58,22 @@ packages:
...
 extra-deps:
 - git: https://github.com/TorXakis/text-via-sockets.git
   commit: e3228cd0407ec0d7991a544e154ea2def184fcae
+- servant-0.13
+- servant-server-0.13
+- servant-multipart-0.11.1
+- servant-docs-0.11.2
+- servant-swagger-1.1.5
+- http-types-0.12.1
+- text-1.2.3.0
```
Comment (pjljvandelaar) [inline stack.yaml]: why are these extra-deps needed?
servant-0.13 is in lts 11.1, like text-1.2.3.0.....

File: sys/txs-lib/license.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, TorXakis
```
Comment (pjljvandelaar) [inline sys/txs-lib/license.txt:3]: Do we need to update to 2018?
Or change license to something like  2017 - 2018 
or just 2017 -

**Review by pjljvandelaar — COMMENTED** [review]
2 minor comments

File: sys/txs-lib/license.txt (hunk 2 above)
Comment (keremispirli) [reply sys/txs-lib/license.txt:3]: Do we need this licence file at all? We already have ours in the [root](https://github.com/TorXakis/TorXakis/blob/feat/txs-interface/LICENSE). It also says 2017 though...

File: stack.yaml (hunk 1 above)
Comment (keremispirli) [reply stack.yaml]: I'll check if it works without them.

Comment (dnadales) [reply stack.yaml]: These imports were needed to be able to use `hoistServer`. Now we're just using the `Handler` provided by servant. We shoudn't need these anymore, so I'd guess they can be safely removed.

File: sys/txs-lenses/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (dnadales) [inline sys/txs-lenses/README.md]: I think this README is not relevant here.

**Pull request closed without merging**